helm/linkerd2-multicluster-app/Chart.yaml

```diff
@@ -1,23 +1,23 @@
-apiVersion: "v1"
-appVersion: stable-2.11.4
+apiVersion: v2
+appVersion: stable-2.12.2
+annotations:
+  application.giantswarm.io/team: team-cabbage
 description: |
   The Linkerd-Multicluster extension contains resources to support multicluster
   linking to remote clusters
-home: https://linkerd.io
+home: https://github.com/giantswarm/linkerd2-multicluster-app
+icon: https://linkerd.io/images/logo-only-200h.png
```

Comment:
Not blocking, did you check if this logo is available in https://github.com/giantswarm/web-assets?

Reply:
there is indeed.